integration_tests/environment.py

```diff
@@ -16,14 +16,15 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 import contextlib
-import shutil
 import os
+import shutil
+from time import sleep
 
 import d3a.constants
+from d3a import constants
```

Comment:
Any specific reason to have both style of imports?

Reply:
Not really, will change it.